src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -153,13 +130,21 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
             <Typography variant="h6">Root dataset:</Typography>
             <Typography
               data-cy="snapshot-source-dataset"
-              className={classes.datasetText}
+              sx={(theme) => ({
+                ...(theme as CustomTheme).mixins.ellipsis,
+              })}
               component="span"
             >
               <Link to={`/datasets/${sourceDataset.id}`}>
-                <span className={classes.jadeLink} title={sourceDataset.name}>
+                <Box
+                  component="span"
+                  sx={(theme) => ({
+                    ...(theme as CustomTheme).mixins.jadeLink,
+                  })}
```

Comment:
```suggestion
                sx={(theme: CustomTheme) => ({
                    ...theme.mixins.jadeLink,
                  })}
```
Would something like this work? It compiled okay in intellij for me, but I'm not convinced I'm seeing all of the typescript errors on my local setup.

Reply:
Intellij does complain for me with the current linting rules. This was one of the `@ts-ignore` issues we sorted out in mobbing.

![Screenshot 2025-02-28 at 3 43 45 PM](https://github.com/user-attachments/assets/23cf3286-9de0-44a5-bef5-bfa6f070cb40)